src/admin/paths/admin@reports@eject_expenses.yaml

```diff
@@ -0,0 +1,49 @@
+post:
+  tags:
+    - Reports
+  summary: Eject expenses from a report
+  description: Eject expenses from a report
+  operationId: report_eject_expenses_post
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          additionalProperties: False
+          required:
+            - data
+          properties:
+            data:
+              $ref: '../../components/schemas/report.yaml#/report_action_with_expenses_ids_to_eject'
```

Comment:
where is this written?

Reply:
not able to find the definition of `report_action_with_expenses_ids_to_eject`

Reply:
Please check

Reply:
report_action_with_expenses_ids_to_eject:
  type: object
  additionalProperties: False
  required:
    - id
    - expense_ids
  properties:
    id:
      type: string
      description: |
        id of report, from which the expenses are to be ejected
      example: 'rpa21A2rdj2'
    expense_ids:
      type: array
      items:
        allOf:
          - $ref: './fields.yaml#/id_string'
      example: [ 'txelntgwgwll' ]

Reply:
https://github.com/fylein/fyle-platform-docs/blob/main/src/components/schemas/report.yaml

![Screenshot 2023-05-23 at 10 25 35 AM](https://github.com/fylein/fyle-platform-docs/assets/57040393/fb275b1e-43fc-418a-802a-dc8f5478fd45)